llvm/test/CodeGen/AArch64/float16-promotion-with-nofp.ll

```diff
@@ -0,0 +1,21 @@
+; RUN: llc -mcpu=cortex-r82 -O1 -o - %s | FileCheck %s
```

Comment:
You can use -mtiple=aarch64 -mattr=-fp-armv8 (and remove -O1).

I would also suggest using update_llc_test_checks.py for generating the run lines too.